packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -444,7 +444,8 @@ export const EVENT_TRIGGERS = {
   MEETINGS_NETWORK_DISCONNECTED: 'network:disconnected',
   MEETINGS_NETWORK_CONNECTED: 'network:connected',
   MEETING_SELF_LEFT: 'meeting:self:left',
-  NETWORK_QUALITY: 'network:quality'
+  NETWORK_QUALITY: 'network:quality',
+  MEDIA_SERVER_NEGOTIATED: 'media:serverNegotiated'
```

Comment:
maybe simply "media:negotiated" would be better? it's just a suggestion, you can keep it as it is.